man/systemd.netdev.xml

```diff
@@ -432,6 +432,17 @@
           </para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>EgressQOSMaps=</varname></term>
+        <listitem>
+          <para>Defines a mapping of Linux internal packet priority (SO_PRIORITY) to VLAN header
```

Comment:
doesn't matter, but maybe enclose `SO_PRIORITY` in `<constant></constant>` or so?